README.rst

```diff
@@ -15,7 +15,7 @@
 
 |
 
-CrateDB is a distributed SQL database that makes it simple to store and analyze massive amounts of machine data in real-time.
+CrateDB is a distributed SQL database which simplifies storing and analyzing massive amounts of machine data in real-time.
```

Comment:
the description has been shortened, but it changes the key verbs from infinitives ("store", "analyze") to gerunds ("storing", "analyzing"). this makes the sentence less direct and also less "formal sounding